.github/workflows/jira-issue-check.yml

```diff
@@ -0,0 +1,21 @@
+name: JIRA Issue Check
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  check-jira-issue:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check JIRA Issue Number
+        id: jira-issue-check
+        run: ./bin/jira-issue-check.sh
```

Comment:
You can't refer local files in shared workflows like this. GH action limitations. 

But you can download it from here, see how sbom action works:
https://github.com/TykTechnologies/github-actions/blob/main/.github/workflows/sbom.yaml#L94